storagenode/piecestore/endpoint.go

```diff
@@ -44,8 +44,8 @@ var _ pb.PiecestoreServer = (*Endpoint)(nil)
 // OldConfig contains everything necessary for a server
 type OldConfig struct {
 	Path string `help:"path to store data in" default:"$CONFDIR/storage"`
-
-	WhitelistedSatelliteIDs string        `help:"a comma-separated list of approved satellite node ids" devDefault:"" releaseDefault:"12EayRS2V1kEsWESU9QMRseFhdxYxKicsiFmxrsLZHeLUtdps3S,118UWpMCHzs6CvSgWd9BfFVjw5K9pZbJjkfZJexMtSkmKxvvAW,121RTSDpyNZVcEU84Ticf2L1ntiuUimbWgfATz21tuvgk3vzoA6,12L9ZFwhzVpuEKMUNUqkaTLGzwY9G24tbiigLiXpmZWKwmcNDDs"`
+	// TODO: add valid formats in description?
+	WhitelistedSatelliteURLs string        `help:"a comma-separated list of approved satellite node urls" devDefault:"" releaseDefault:"12EayRS2V1kEsWESU9QMRseFhdxYxKicsiFmxrsLZHeLUtdps3S@mars.tardigrade.io:7777,118UWpMCHzs6CvSgWd9BfFVjw5K9pZbJjkfZJexMtSkmKxvvAW@satellite.stefan-benten.de:7777,121RTSDpyNZVcEU84Ticf2L1ntiuUimbWgfATz21tuvgk3vzoA6@saturn.tardigrade.io:7777,12L9ZFwhzVpuEKMUNUqkaTLGzwY9G24tbiigLiXpmZWKwmcNDDs@jupiter.tardigrade.io:7777"`
```

Comment:
Use NodeURL as type

Reply:
Do I need to add a case to `bindConfig` for NodeURLs?

Reply:
I think it should work as is.

Reply:
Ah nevermind, it has hardcoded all the different scenarios rather than relying on flag interface.

Reply:
The implementation should be the same as `memory.Size` and add it for `storj.NodeURL` and `storj.NodeURLs`.

Reply:
I'm having some trouble adding `storj.NodeURL` to the test config in `bind_test.go`. `storj.NodeURLs` appears to be fine, but when I add `storj.NodeURL` bindConfig panics at 
```func bindConfig(flags FlagSet, prefix string, val reflect.Value, vars map[string]confVar, setupCommand, setupStruct bool, isDev bool) {
	if val.Kind() != reflect.Struct {
		panic(fmt.Sprintf("invalid config type: %#v. Expecting struct.", val.Interface()))
	}```

Reply:
made a separate fix for it https://github.com/storj/storj/pull/2425

Reply:
made a separate fix for it https://github.com/storj/storj/pull/2425